src/structs/pdb.rs

```diff
@@ -1051,6 +1051,26 @@ impl<'a> PDB {
         }
         chains
     }
+
+    /// Returns a vector of unique residue names present in the PDB file.
+    ///
+    /// # Arguments
+    ///
+    /// * `self` - A reference to the PDB struct.
+    ///
+    /// # Returns
+    ///
+    /// * `Vec<String>` - A vector of unique residue names.
+    pub fn residues_list(&self) -> Vec<String> {
```

Comment:
I would propose to rename this function to `unique_residue_names` although it is a bit longer it captures the behaviour a bit better I think.